include/librealsense2/h/rs_types.h

```diff
@@ -119,7 +119,8 @@ typedef struct rs2_pose
 
 /** \brief Severity of the librealsense logger. */
 typedef enum rs2_log_severity {
-    RS2_LOG_SEVERITY_DEBUG, /**< Detailed information about ordinary operations */
+    RS2_LOG_SEVERITY_ALL, /**< Include any/all log messages */
```

Comment:
I'd strongly prefer to add at the end of the enum, not start

Reply:
Done